smartlead/metadata/schemas.json

```diff
@@ -0,0 +1,95 @@
+{
+  "data": {
+    "campaigns": {
+      "displayName": "Campaigns",
+      "fields": {
+        "id": "id",
+        "user_id": "User id",
+        "created_at": "Create at",
```

Comment:
This should be Created at, and `User id` can be `User ID`, same with client ID. Please update this wherever applicable for all objects (uppercase for acronyms)

Reply:
I was talking with Lauren the other day about ID conventions in golang code. And to be safe I made it lower case here.
Maybe for display names it is fine.